api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMERSemiAnnualReport.java

```diff
@@ -60,8 +60,9 @@ public ReportDefinition constructReportDefinition() {
     rd.setName(getName());
     rd.setDescription(getDescription());
     rd.setParameters(txMlDataset.getParameters());
-    rd.addDataSetDefinition("TXML", Mapped.mapStraightThrough(txMlDataset.constructtxMlDataset()));
-    rd.addDataSetDefinition("T", Mapped.mapStraightThrough(txTBDataset.constructTxTBDataset()));
+    // rd.addDataSetDefinition("TXML",
+    // Mapped.mapStraightThrough(txMlDataset.constructtxMlDataset()));
+    // rd.addDataSetDefinition("T", Mapped.mapStraightThrough(txTBDataset.constructTxTBDataset()));
```

Comment:
@cfaife @psmatsinhe You'll uncomment this right?

Reply:
yes sure